fe/fe-core/src/main/java/org/apache/doris/resource/Tag.java

```diff
@@ -69,10 +69,13 @@ public class Tag implements Writable {
     public static final String CLOUD_CLUSTER_NAME = "cloud_cluster_name";
     public static final String CLOUD_CLUSTER_ID = "cloud_cluster_id";
     public static final String CLOUD_UNIQUE_ID = "cloud_unique_id";
-    public static final String CLOUD_CLUSTER_PUBLIC_ENDPOINT = "cloud_cluster_public_endpoint";
-    public static final String CLOUD_CLUSTER_PRIVATE_ENDPOINT = "cloud_cluster_private_endpoint";
     public static final String CLOUD_CLUSTER_STATUS = "cloud_cluster_status";
 
+    public static final String COMPUTE_GROUP_PUBLIC_ENDPOINT = "compute_group_public_endpoint";
+    public static final String COMPUTE_GROUP_PRIVATE_ENDPOINT = "compute_group_private_endpoint";
```

Comment:
we should do some investigation before changing it
# there are some existed tags named "cloud_cluster_public_endpoint"
# there may be some managements procedure relies on the name "cloud_cluster_public_endpoint"

Reply:
The tag name is not updated here, it only changed a variable name.